deploy/kubernetes/kustomization/image_setup/alpha/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
naming nit: we have more than just image version patches in these directories. We may enable new features as well.

Maybe I would still keep the directory name as "overlays". So the main change is that the node deployment is separated from controller deployment, but we still use the kustomization.yaml from the "overlays" directory to determine which version of the driver to deploy.

Reply:
updated.